sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -341,6 +343,96 @@ _nib_dr_entry_t *_nib_drl_get_dr(void)
     return _prime_def_router;
 }
 
+_nib_offl_entry_t *_nib_offl_alloc(const ipv6_addr_t *next_hop, unsigned iface,
+                                   const ipv6_addr_t *pfx, unsigned pfx_len)
+{
+    _nib_offl_entry_t *dst = NULL;
+
+    assert((pfx != NULL) && (!ipv6_addr_is_unspecified(pfx)) &&
+           (pfx_len != 0) && (pfx_len <= 128));
```

Comment:
just taste, but I like `pfx_len > 0` more, because it's in harmony with `px_len <= 128`